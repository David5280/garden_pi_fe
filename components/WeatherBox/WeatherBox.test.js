import React from 'react';
import { WeatherBox } from './index';
import 'react-native';
import renderer from 'react-test-renderer';


test('renders correctly', () => {
  const tree = renderer.create(<WeatherBox />).toJSON();
  expect(tree).toMatchSnapshot();
});