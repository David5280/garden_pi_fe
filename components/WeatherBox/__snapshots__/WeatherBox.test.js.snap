// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#d5fdd5",
      "borderColor": "#A14550",
      "borderRadius": 30,
      "borderWidth": 2,
      "marginLeft": 0,
      "marginRight": 2,
      "padding": 2,
      "width": 637.5,
    }
  }
>
  <Text
    style={
      Object {
        "fontSize": 18,
        "fontWeight": "bold",
        "marginTop": 5,
      }
    }
  />
  <View
    style={
      Object {
        "display": "flex",
        "flexDirection": "row",
        "justifyContent": "space-evenly",
        "marginBottom": 10,
      }
    }
  >
    <Text
      style={
        Object {
          "fontSize": 28,
          "fontWeight": "bold",
          "marginLeft": 20,
          "marginTop": 20,
        }
      }
    >
      NaN
      °F / 
      NaN
      °F
    </Text>
  </View>
  <Text
    style={
      Object {
        "fontSize": 18,
      }
    }
  />
</View>
`;
